omod/src/main/java/org/openmrs/module/coreapps/page/controller/clinicianfacing/PatientPageController.java

```diff
@@ -94,6 +94,21 @@ else if (patient.isVoided() || patient.isPersonVoided()) {
         AppContextModel contextModel = sessionContext.generateAppContextModel();
         contextModel.put("patient", new PatientContextModel(patient));
         contextModel.put("visit", activeVisit == null ? null : new VisitContextModel(activeVisit));
+
+        List<EncounterType> encounterTypes = new ArrayList<EncounterType>();
+        List<Encounter> encounters = encounterService.getEncountersByPatient(patient);
```

Comment:
You seem to have done changes for two tickets in one pull request. Can you separate them?